superset/common/query_object.py

```diff
@@ -61,6 +61,7 @@ class DeprecatedField(NamedTuple):
     new_name: str
 
 
+# Since we are getting rid of druid we are not replacing it with a new field. Should this be updated?
```

Comment:
is this comment referring to druid_time_origin? If so, I'd say yes we can remove it.

Reply:
Thank you